### PR TITLE
Add LBRY specific operations to lbry-explorer

### PR DESCRIPTION
**Pull request by loblao**
As on https://lbry.io/bounty/modified-block-explorer

It's now possible to see associated claims and its supports with a transaction. It's also possible to lookup claim history for a name.

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -36,13 +37,22 @@ function route_get_block(res, blockhash) {
 /* GET functions */
 
 function route_get_tx(res, txid) {
+  function get_claim_info_then_render(info)
+  {
+      uri = 'http://127.0.0.1:' + settings.port + '/ext/claims/tx/' + txid;
```
Comment (lyoshenka) [inline routes/index.js]: why is this making requests to localhost? cant it just make a direct call to get whatever it needs?

**Comment by lyoshenka** [comment]
Sorry for the slow review. There are two things missing here:

1. Error handling. For example https://explorer.lbry.io/api/getclaimsforname says "there was an error". It should tell you what the error is, how you can fix it, etc. Same for other endpoints

2. Docs. Please add the new endpoints to https://explorer.lbry.io/info

**Comment by lyoshenka** [comment]
Another unhandled case: https://explorer.lbry.io/api/getclaimsfortx?txid=427b

It should tell you the txid is invalid.

**Comment by lyoshenka** [comment]
An error from the logs:

```
/home/lbry/explorer/lib/explorer.js:427
          body.forEach(function(claim)
              ^

TypeError: Cannot read property 'forEach' of undefined
    at Request._callback (/home/lbry/explorer/lib/explorer.js:427:15)
    at Request.self.callback (/home/lbry/explorer/node_modules/request/request.js:121:22)
    at emitTwo (events.js:87:13)
    at Request.emit (events.js:172:7)
    at Request.<anonymous> (/home/lbry/explorer/node_modules/request/request.js:985:14)
    at emitOne (events.js:82:20)
    at Request.emit (events.js:169:7)
    at IncomingMessage.<anonymous> (/home/lbry/explorer/node_modules/request/request.js:936:12)
    at emitNone (events.js:72:20)
    at IncomingMessage.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:921:12)
    at nextTickCallbackWith2Args (node.js:442:9)
    at process._tickCallback (node.js:356:17)
```

**Comment by lyoshenka** [comment]
https://explorer.lbry.io/ext/claims/tx/2342b

**Comment by loblao** [comment]
Sorry for my delay. Apparently those issues have been fixed in 0ce68de6c110d8d8d8d449f3d941410299c811ad (thanks!), is there anything else I can help with?

**Pull request merged**